docs/contributor/04-30-ci-cd.md

```diff
@@ -69,8 +69,7 @@ The following CI jobs are part of the release cycle.
 | [`UI Tests / Run UI tests`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/ui-tests.yaml#L23)                                                        | Runs UI tests for Busola extensions of API Gateway module and API Rules                                                                                                    |
 | [`PR Integration Release / Custom domain GCP`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/pull-integration-release.yaml#L88)                     | Executes the integration test suite that verifies the functional correctness of Kyma API Gateway Operator. The test suite uses a custom domain and a Gardener GCP cluster. |
 | [`PR Integration Release / Custom domain AWS`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/pull-integration-release.yaml#L107)                    | Executes the integration test suite that verifies the functional correctness of the API Gateway Operator. The test suite uses a custom domain and a Gardener AWS cluster.  |
-
-
+| [`PR Build and Push Manager Image / Build and push manager image`](https://github.com/kyma-project/api-gateway/blob/8b3eb837f1474c2e35eb592afbc2e86f69b963f4/.github/workflows/pull-build-push.yaml#L44) | Builds the managers's image and pushs it to a GCP Artifact Registry so that the jobs running tests on Gardener clusters can be executed.               |
```

Comment:
```suggestion
| [`PR Build and Push Manager Image / Build and push manager image`](https://github.com/kyma-project/api-gateway/blob/8b3eb837f1474c2e35eb592afbc2e86f69b963f4/.github/workflows/pull-build-push.yaml#L44) | Builds the manager's image and pushes it to a GCP Artifact Registry so that the jobs running tests on Gardener clusters can be executed.               |
```